src/main/kotlin/no/nav/tilleggsstonader/klage/behandling/BehandlingService.kt

```diff
@@ -104,10 +104,6 @@ class BehandlingService(
             "Påklaget vedtak er i en ugyldig tilstand: EksternFagsystemBehandlingId:${påklagetVedtakDto.eksternFagsystemBehandlingId}, PåklagetVedtakType: ${påklagetVedtakDto.påklagetVedtakstype}"
         }
 
-        feilHvis(påklagetVedtakDto.manglerVedtaksDato()) {
-            "Må fylle inn vedtaksdato når valgt vedtakstype er ${påklagetVedtakDto.påklagetVedtakstype}"
-        }
```

Comment:
Fjernes muligheten for å sette vedtaksdato manuellt eller hvorfor fjernes dette?